panc/src/main/scripts/panlint/panlint.py

```diff
@@ -180,7 +180,7 @@ def merge_diagnoses(args):
         return ''
 
     args = [a.rstrip() for a in args]
-    result = [' '] * max(map(len, args))
+    result = [' '] * max([len(a) for a in args])
```

Comment:
The code is equivalent, but:
1. Pylint discourages use of `map()` and `filter()`.
2. It may make Python3 transition easier as `map()` has a different behaviour in Python3.
3. It may be easier to read.